### DISCUSSION TITLE
How to change the line height of a font

### DISCUSSION THREAD

**Discussion by rajputrajat** [wezterm/wezterm#387] [Q&A] [answered]
"Hack NF" is my choice of font.
but it's height is rather less, rendering text not very readable.
How do I configure it?

**Comment by wez** [comment, marked as answer]
I've just pushed 0ae494fb94ab0bd3682c680673d691b9cd31f373 which adds a new line_height option:

```lua
return {
   -- Scale the default line height by 1.2, making it 20% larger than
   -- the value specified by the font author
   line_height = 1.2,
}
```

That should start showing up in nightly builds within a couple of hours.

**Reply by rajputrajat** [reply]
are you wez or a wiz?
Thanks a lot.
==
tested with latest nightly, this new config option is working fine.
